lonestar/analytics/cpu/louvain_clustering/louvain_clustering_cli.cpp

```diff
@@ -91,10 +91,11 @@ main(int argc, char** argv) {
   totalTime.start();
 
   if (!symmetricGraph) {
```

Comment:
same here, I think it is better to show the following warning if the flag is set.